contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -490,7 +490,7 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5 {
     }
 
     // return list campaign ids for epoch, excluding non-existed ones
-    function getListCampIDs(uint256 epoch) external view returns (uint256[] memory campaignIDs) {
+    function getListCampaignIDs(uint256 epoch) external view returns (uint256[] memory campaignIDs) {
```

Comment:
is this API supported on Jarvis?

should consult Victor for changing this

I support this change!

Reply:
Victor has not updated Jarvis with newest APIs for submit campaign, get campaign details. So he will update Jarvis once we finalize

Reply:
OK
Great